chsdi/models/vector/__init__.py

```diff
@@ -79,19 +82,17 @@ def __read__(self):
                     if hasattr(self, '_shape'):
                         geom = self._shape
                     elif val is not None:
-                        if len(val.data) > 1000000:
-                            raise HTTPBandwidthLimited(
-                                'Feature ID %s: is too large' % self.id)
                         geom = to_shape(val)
```

Comment:
Wouldn't it be better to put it after the length test ? Seems to me like it would transform the data into a shape, then verify if the data isn't too big to put into a shape. As to_shape(val) is the expected return value, it would be definitiely be better to do a 
```
if len(val.data) > MAX_FEATURE_GEOMETRY_SIZE:
  geom = None
else: 
  geom = to_shape(val)
```